src/utils/positionpnl/positionpnl.js

```diff
@@ -0,0 +1,66 @@
+import {
+  retrieveData,
+  insertData,
+  putData
+} from '../network'
+import PositionPNL from '../../transactions/PostionPNL/PositionPNL'
+
+export function retrieve({ AMId }, callback) {
```

Comment:
I think the API accepts `businessDate` and `bookIds` as query params.

Reply:
Updated

Reply:
You'll also need to add them in the argument.